pyaedt/modules/SetupTemplates.py

```diff
@@ -667,7 +668,8 @@ def HFSS3DLayout_AdaptiveFrequencyData(freq):
 
 
 class SweepHFSS(object):
-    """ """
+    """SweepHFSS class.
+    """
```

Comment:
```suggestion
    """SweepHFSS class."""
```
Single sentences can be opened and closed with `"""` on the same line.